src/dotnet/Chat.UI.Blazor/Components/ContactLinks.razor

```diff
@@ -18,19 +18,32 @@
 @code {
     public class Model
     {
-        public UserContact[] Contacts { get; set; } = new UserContact[0];
+        public (UserContact, string)[] Contacts { get; set; } = new (UserContact, string)[0];
     }
 
     [Inject] private Session Session { get; set; } = null!;
     [Inject] private IUserContacts Contacts { get; set; } = null!;
+    [Inject] private IChatAuthors ChatAuthors { get; set; } = null!;
 
     protected override ComputedState<Model>.Options GetStateOptions()
         => new() { InitialValue = new () };
 
     protected override async Task<Model> ComputeState(CancellationToken cancellationToken) {
         var contacts = await Contacts.GetContacts(Session, cancellationToken).ConfigureAwait(false);
+        var list = new List<(UserContact, string)>();
+        foreach (var contact in contacts.OrderBy(c => c.Name)) {
+            string url = contact.Id;
+            var isAuthorsPeerChat = contact.TargetUserId.Value.Contains(":");
+            if (isAuthorsPeerChat) {
```

Comment:
isn't TryParse meant to check that it's a peer chat along with parsing? like we do it for int or decimal? ```if(int.TryParse....)```